modules/pam_faillock/pam_faillock.8.xml

```diff
@@ -327,6 +327,12 @@ session  required       pam_selinux.so open
         <term><filename>/var/run/faillock/*</filename></term>
         <listitem>
           <para>the files logging the authentication failures for users</para>
+	  <para>
+            Note: User specific files will not be available after reboot on system’s configured
+            with directory <filename>/var/run/faillock</filename> mounted on virtual memory.
+            For persistent storage use the option <emphasis>dir=</emphasis> in
+            file <filename>/etc/security/faillock.conf</filename>. 
+	  </para>
```

Comment:
Fix indentation

Reply:
Hi Iker, 

Thank you for your feedback.

I have made the necessary changes.

Can you kindly review again. 

Thank you!

Reply:
Now it looks good to me.